test/image/mocks/waterfall_profit-loss_2018vs2019_textinfo_base.json

```diff
@@ -123,6 +123,12 @@
         }
     ],
     "layout": {
+        "font": {
```

Comment:
@archmoj probably already on your radar but would be nice to try some of these mocks with different combinations of font properties

Reply:
I agree, it should at least be different from the default values.